Gemfile

```diff
@@ -5,7 +5,6 @@ source 'https://rubygems.org'
 gemspec path: '.'
 
 # use ENV vars, with default value as fallback for local setup
-ruby(ENV['RUBY_VERSION'] || '3.2.2')
```

Comment:
I guess for a gem it is not as useful as for an app. but this ensure the app runs on the ruby it is supposed to.

people running in docker its usually not a problem, but systems with multiple ruby, `bundler` will complain if the ruby does not have the right version.

I don't have an objection if you prefer removing it.

Reply:
Indeed, for an app that doesn't have a gemspec it can make sense but for a gem I don't see the point.

Reply:
in a way, it was adding a guarantee that GHA, indeed, runs the test with the correct ruby version.

I don't remember if this project is one of the first that I moved from multiple Gemfiles to a single. but it might be the reason.